target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -22,6 +22,11 @@ ath79_setup_interfaces()
 	mikrotik,routerboard-wap-g-5hact2hnd)
 		ucidef_set_interface_lan "eth0"
 		;;
+	mikrotik,routerboard-952ui-5ac2nd)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
```

Comment:
ucidef_set_interface_wan "eth1"
is enough, lan is set via switch statement below.